codes/javascript/chapter_computational_complexity/time_complexity.js

```diff
@@ -0,0 +1,159 @@
+/**
+ * File: time_complexity.js
+ * Created Time: 2023-01-02
+ * Author: RiverTwilight (contact@rene.wang)
+ */
+
+class time_complexity {
```

Comment:
We don't have to create a class for these functions in JS and TS. Please refer to other JS and TS codes to see the details.

Reply:
So sorry for this. I made this decision as the Java code does. Because the contribution guide recommended doing so.